### PR TITLE
feat: design tweaks

### PR DESCRIPTION
**Pull request by d-Rickyy-b**
I changed the calendar day boxes to use a wider layout and I centred the calendar module header.

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -11,9 +11,15 @@
...
 
+.module.MMM-MonthCalendar .module-header {
+    text-align: center;
+}
+
 .module.MMM-MonthCalendar .calendar-daysinweek div,
 .module.MMM-MonthCalendar .calendar-days div {
     padding: 4px 0;
+    margin-left: 8px;
+    margin-right: 8px;
```
Comment (PalatinCoder) [inline main.css]: I prefer to write `margin: 0 8px`. Looks a bit smoother IMHO 😉

File: main.css [hunk 2]
```diff
@@ -11,9 +11,15 @@
     font-weight: 600;
 }
 
+.module.MMM-MonthCalendar .module-header {
+    text-align: center;
+}
+
```
Comment (PalatinCoder) [inline main.css]: Not center-aligning the header was actually intentional, to keep it inline with the other modules. Depending on the region where the module is displayed, the header will be aligned accordingly by the global css. IMO users should override this behavior in their custom.css if they want to do so. WDYT?

**Review by PalatinCoder — CHANGES_REQUESTED** [review]
Left some comments in the code 😉

File: main.css (hunk 2 above)
Comment (d-Rickyy-b) [reply main.css]: I removed the centering to adhere to the global style

File: main.css (hunk 1 above)
Comment (d-Rickyy-b) [reply main.css]: Looks indeed better :)

**Review by PalatinCoder — APPROVED** [review]
👍

**Pull request merged**